media/src/objects/Curve.svelte

```diff
@@ -460,6 +460,7 @@
                         frame.visible = true;
                         animation = !animation;
                         if (animation) dispatch("animate");
+                        render();
```

Comment:
This change seems unrelated to the keyboard stuff - let's address this on a separate branch.